assets/js/blocks/price-filter/block.js

```diff
@@ -12,13 +12,40 @@ import PriceSlider from '@woocommerce/base-components/price-slider';
 import { useDebouncedCallback } from 'use-debounce';
 import PropTypes from 'prop-types';
 import { getCurrencyFromPriceResponse } from '@woocommerce/price-format';
+import { getSetting } from '@woocommerce/settings';
 
 /**
  * Internal dependencies
  */
 import usePriceConstraints from './use-price-constraints.js';
 import './style.scss';
 
+function findGetParameter( paramName ) {
+	if ( ! window ) {
+		return;
+	}
+	const url = new URL( window.location );
+	const params = new URLSearchParams( url.search );
+
+	return params.get( paramName );
+}
+
+function formatParams( url, paramSettings ) {
+	const params = new URLSearchParams( url.search );
+	const keys = Object.keys( paramSettings );
+
+	for ( let i = 0; keys.length > i; i++ ) {
+		const { value, constraint } = paramSettings[ keys[ i ] ];
+		if ( value === constraint ) {
+			params.delete( keys[ i ] );
+		} else {
+			params.set( keys[ i ], value / 100 );
+		}
+	}
```

Comment:
```suggestion
    for ( const [ key, setting ] of Object.entries( paramSettings ) ) {
        const { value, constraint } = setting;
 
		if ( value === constraint ) {
			params.delete( key );
		} else {
			params.set( keys, value / 100 );
		}
	}
```

or, of course, we could destructure that in the loop declaration itself:

```js
    for ( const [ key, { value, constraint } ] of Object.entries( paramSettings ) ) {
```

Either way, I think using `for ... of` is more idiomatic than a regular `for` loop.